pkg/cmd/extension/command.go

```diff
@@ -220,6 +246,26 @@ func checkValidExtension(rootCmd *cobra.Command, m extensions.ExtensionManager,
 	return nil
 }
 
+func isBinExtension(client *http.Client, repo ghrepo.Interface) (isBin bool, err error) {
+	hs, err := hasScript(client, repo)
+	if err != nil || hs {
+		return
```

Comment:
So if the git repository happens to have a `gh-my-extension` file in its root, this will assume that it's a regular extension and not a binary extension and won't even look up releases?

Couldn't we just look up releases anyway and decide that it's a binary extension if it has binary attachments?

Reply:
we could i think. I had some reason for doing the script check that I don't think I recorded anywhere. I don't think I was thinking about strictly looking at the asset names. I'll switch to that and see how it feels.